# Influenza A H3N2 virus phylogeography, Kilifi, Kenya, 2015-2016

Author:	David Collins Owuor

Institution:	KEMRI-Wellcome Trust Research Programme, Kilifi, Kenya

Date Published: 30 August 2020

## Introduction

This repository contains data and input files for the phylogeographical analysis of **influenza A H3N2 virus from Kilifi, Kenya, 2015-2016.**

## General instructions

The data and instructions will allow replication of the results shown in this work.

## Data

This folder contains data and input files for all steps of the phylogenetic and phylogeographical analyses,
including:

* the multiple sequence alignment fasta file with Kilifi and global genomes from GISAID in H3N2_Kilifi_and_global_genomes_2014-2016;
* the output of the maximum-likelihood phylogenetic trees (ultrafast bootstrap) in H1N1pdm09_Kenya_and_global_genomes_ML_trees;
* the result of the ancestral state reconstruction (mugration model) in H3N2_Kilifi_and_global_ancestral_state_reconstruction;   
* the xml file used for the phylogeographic analysis with BEAST stating all parameters in H3N2_Kilifi_phylogeography;
* the edited trees file from BEAST analysis for BaTS analysis in **H3N2_Kilifi_BaTS**.

##	Figures

This folder contains figures from the analyses.
