# Influenza A H3N2 virus phylogeography, Kilifi, Kenya, 2015-2016

Author:	David Collins Owuor

Institution:	KEMRI-Wellcome Trust Research Programme, Kilifi, Kenya

Date Published: 30 August 2020


## Introduction

This repository contains data and input files for the phylogeographical analysis of
**influenza A H3N2 virus from Kilifi, Kenya, 2015-2016.**


## General instructions

The data and instructions will allow replication of the results in my PhD thesis available
at http://oro.open.ac.uk/75472/; Chapter 3 - Phylogeography of Influenza A(H3N2) Virus In
Kilifi, Kenya, 2015-2016.


## Datasets

This folder contains next generation sequencing (NGS) data and input files for the
phylogenetic and phylogeographical analyses.

1.	1_H3N2_Kilifi_and_global_genomes_2014-2016

*	H3N2_Kilifi_Kenya_2015-2016_genome_details.txt 

A list of 58 H3N2 virus (2015-2016) NGS data from Kenya showing: strain - virus strain
identity; accession - GISAID accession number; site - location of isolation; reads - number
of sequencing reads; number of influenza virus reads; Ct - PCR Ct value; date - collection
dates; and strain_accession - strain ID showing collection site for analysis.


*	H3N2_global_2014-2016_genome_details.txt

A list of global influenza H3N2 virus data from 2014-16 showing: strain - virus strain
identity; accession - GISAID accession number; country - country of isolation; and
continent - continent of isolation.


2.	2_H3N2_Kilifi_phylogeography

File with KHDSS location coordinates and geojson file for phylogeographical analyses of
Kilifi virus sequence data.

* kilifi_coodinates.txt - Kilifi coordinates;

* kilifi.geojson - Kilifi geojson file for phylogeographical analysis of Kilifi NGS data.


3.	3_H3N2_Kilifi_BaTS
  
The edited trees file from BEAST analysis for BaTS analysis.

*	H3N2_Kilifi_2015-2016_WGS_GTRG_UncorrRelax_LogNorm_Bayesian_SkyGrid_BSSVS_400M_bats_locations.trees


##	Figures

This folder contains figures from the analyses.
