# Influenza A H3N2 virus phylogeography, Kilifi, Kenya, 2015-2016

Author:	David Collins Owuor

Institution:	KEMRI-Wellcome Trust Research Programme, Kilifi, Kenya

Date Published: 30 August 2020

# Analysis

This folder contains analysis scripts and analysis files used to generate chapter figures.

1.	1_KHDSS_site_frequency

*	Site_frequency.R - R script for plotting sampled influenza viruses over time. 

*	KHDSS_site_frequency.csv - File with data on sampled influenza viruses.

*	Figure_3.1.pdf

2. 2_KHDSS_site_sequencing

*	Site_sequencing.R - R script for plotting sequenced influenza A(H3N2) viruses over time.

*	KHDSS_site_sequencing.csv - File with data on sequenced influenza viruses.

*	Figure_3.2.pdf

3. 3_KHDSS_sequencing_by_location

*	Genetic_groups_by_location.R - R script for plotting genetic groups of A(H3N2) viruses
by KHDSS location.

*	Genetic_groups_by_location.csv - File with data on genetic groups of A(H3N2) viruses.

*	Figure_3.3.pdf

4. 4_Tanglegrams

* meta.json and tree.json files of 8 gene segments for each A(H3N2) virus genome from
Kilifi for visualization of tanglegrams in Nextstrain (https://nextstrain.org) to identify
reassortment events in A(H3N2) viruses.

5. 5_Lineage_annotation

* KHDSS_lineages.R - R script for plotting viral lineages of A(H3N2) viruses by gene
sequence to identify reassortant strains.

* Lineage_metadata.txt - File with data on viral lineages of A(H3N2) viruses from Kilifi.  
